vagrant-centos-kubeadm-v1.9.2/config.yaml

```diff
@@ -0,0 +1,10 @@
+configs:
+    use: 'SA18Lab'    
+    SA18Lab:
```

Comment:
TIL about config.yaml. That's great stuff. What does SA in SA18Lab stand for?

Reply:
Not sure... that's our internal lab :-)